Content.Server/Chat/ChatSystem.cs

```diff
@@ -180,6 +180,8 @@ public void DispatchStationAnnouncement(EntityUid source, string message, string
         var station = _stationSystem.GetOwningStation(source);
         var filter = Filter.Empty();
 
+        filter.AddInRange(Transform(source).MapPosition, 500f);
+
```

Comment:
This is very ugly, I think you're better off reverting this for now and leaving the rest of the fixes.